aula-3/src/main/java/org/acelera/saopaulo/soccer/controller/JogadorController.java

```diff
@@ -0,0 +1,41 @@
+package org.acelera.saopaulo.soccer.controller;
+
+import org.acelera.saopaulo.soccer.dto.JogadorDto;
+import org.acelera.saopaulo.soccer.service.JogadorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/jogador")
+public class JogadorController {
+
+    @Autowired
+    private JogadorService jogadorService;
+
+    @PostMapping("/adiciona")
```

Comment:
vc não precisa criar um path para adicionar o um jogador. 
Lembre-se que vc pode utilizar o verbo post com a mesma URL jogador.
Esse é o nível dois do glory of rest:

http://www.boaglio.com/index.php/2016/11/03/modelo-de-maturidade-de-richardson-os-passos-para-a-gloria-do-rest/